README.rst

```diff
@@ -106,13 +121,30 @@ to find uncovered code.
 .. _`Code of Conduct`: https://www.mdanalysis.org/pages/conduct/
 .. _trajectory formats: https://docs.mdanalysis.org/documentation_pages/coordinates/init.html#id1
 .. _topology formats: https://docs.mdanalysis.org/documentation_pages/topology/init.html#supported-topology-formats
-.. _Issue 87: https://github.com/MDAnalysis/mdanalysis/issues/87
 .. _MDAnalysis: https://www.mdanalysis.org
-.. _LICENSE: https://github.com/MDAnalysis/mdanalysis/blob/master/LICENSE
-.. _`#286`: https://github.com/MDAnalysis/mdanalysis/issues/286
+.. _LICENSE:
+   https://github.com/MDAnalysis/mdanalysis/blob/master/LICENSE
+.. _`Installation Quick Start`:
+   https://www.mdanalysis.org/pages/installation_quick_start/
 .. _`MDAnalysis.analysis`: https://docs.mdanalysis.org/documentation_pages/analysis_modules.html
-.. _`tutorials`: https://www.mdanalysis.org/pages/learning_MDAnalysis/
-.. _`guide`: https://github.com/MDAnalysis/mdanalysis/wiki/Guide-for-Developers
+.. _`tutorials`: https://www.mdanalysis.org/UserGuide/examples/README.html
```

Comment:
I realise it points in the same place, but for consistency should this be: https://userguide.mdanalysis.org/1.0.0/examples/README.html instead?

Reply:
Yes, thanks, or rather https://userguide.mdanalysis.org/examples/README.html (so that we don't have to update the version info)